openfisca_us/parameters/hhs/fpg.yaml

```diff
@@ -1,44 +1,48 @@
 description: Federal poverty guidelines published by the Department of Health and Human Services.
 first_person:
-  CONTIGUOUS_US:
+  metadata:
+    breakdown: state_group
+  Contiguous US:
     2017-01-01: 12_060
     2018-01-01: 12_140
     2019-01-01: 12_490
     2020-01-01: 12_760
     2021-01-01: 12_880
     2022-01-12: 13_590
-  AK:
+  Alaska:
```

Comment:
Do we have to use values instead of codes for this to work? I prefer the conciseness of codes.